contracts/Auctioneer.sol

```diff
@@ -29,7 +29,11 @@ contract Auctioneer is CloneFactory {
     // Holds the address of the auction contract
     // which will be used as a master contract for cloning.
     address public masterAuction;
-    mapping(address => bool) public openAuctions;
+
+    // Maps auction to the item being on offer for that auction (e.g. a tBTC
+    // deposit). Must be filled by the risk manager after `createAuction` was
+    // called.
+    mapping(address => address) public openAuctions;
```

Comment:
Why don't we clean up this mapping on auction regular/early close? We used to delete the mapping from the former `auctionToDeposit` map.